### ISSUE TITLE
OSX source build failing on unloadNamespace

### ISSUE THREAD

**Issue by jennybrown8** [oscarperpinan/rastervis#28]
I'm building rasterVis 0.37 from source on OS X 10.10.4 for use with R 3.1.3.  I'm getting an error during the source build validation step.  I believe this worked correctly in previous versions of rasterVis.

From rasterVis_0.37.tar.gz built with an automated script:

    * installing *source* package ‘rasterVis’ ...
    ** package ‘rasterVis’ successfully unpacked and MD5 sums checked
    ** R
    ** inst
    ** preparing package for lazy loading
    ** help
    *** installing help indices
    ** building package indices
    ** testing if installed package can be loaded
    Error in unloadNamespace(package) : 
      namespace ‘lattice’ is imported by ‘sp’ so cannot be unloaded
    Error in library(pkg, character.only = TRUE, logical.return = TRUE, lib.loc = lib.loc) : 
      “lattice” version 0.20.33 cannot be unloaded.
    Error: loading failed

**Comment by oscarperpinan** [comment]
I don't know why you get this error, and I can't reproduce it (I don't have a OS X computer). The package is being built correctly in the CRAN system (see [check results](https://cran.r-project.org/web/checks/check_results_rasterVis.html)), although they use R 3.2.0.

**Comment by jennybrown8** [comment]
My apologies for taking so long to reply; I was working on the build
environment where I'm building this package, among others.

Today I figured out the cause of my error.  I had a (bad/invalid) saved R
session in my home directory.  Further attempts to build packages silently
loaded that bad session and that caused the namespace-unload-failed error.

The solution for it was:

rm ~/.R*

after which point rasterVis and everything else built correctly.

Sorry for the false alarm, and thank you for looking into it.  Maybe the
solution I've shared will help someone else in the future :)
